modules/cluster-and-ha-management/pages/ha-for-application-server.adoc

```diff
@@ -27,10 +27,10 @@ You can increase the number of servers through the `GUI.BasicConfig.Nodes` confi
 
 The value of `GUI.BasicConfig.Nodes` is a list of nodes that run an application server:
 
-[,console]
+[source.wrap,console]
 ----
 $ gadmin config get GUI.BasicConfig.Nodes
-[{"HostID":"m1","Partition":0,"Replica":1},{"HostID":"m2","Partition":0,"Replica":2},{"HostID":"m3","Partition":0,"Replica":3},{"HostID":"m4","Partition":0,"Replica":4}]
+[{"HostID":"m1","Partition":0,"Replica":1},{"HostID":"m2","Partition":0,"Replica":2},{"HostID":"m3","Partition":0,"Replica":3}]
```

Comment:
@victorleeTG  Thanks for making this change but this brought my attention to a statement in line 25 of this document `ha-for-application-server.adoc`

```
The application server (`gui`) runs on the first three nodes in a cluster by default.
```

This is incorrect. For a cluster with 5 or fewer nodes, `GUI` would run on all the machines in the cluster.
If a cluster has more than 5 nodes, `GUI` would run on the first five nodes. This is what actually happens.

Thanks for making this revision to make it aligned with what we said in line 25 but line 25 in itself is incorrect.